packages/preact/src/index.ts

```diff
@@ -144,10 +160,11 @@ hook(OptionsTypes.RENDER, (old, vnode) => {
 
 		updater = component._updater;
 		if (updater === undefined) {
-			component._updater = updater = createUpdater(() => {
-				component._updateFlags |= HAS_PENDING_UPDATE;
-				component.setState({});
-			});
+			// @ts-expect-error checking symbol in object
+			const isText = !!vnode.type?.[__isTextSymbol];
```

Comment:
let's keep it as before but just perform a condition around the useMemo

Reply:
fixed